services/publish-sketchfab/sketchfab-service.py

```diff
@@ -84,60 +84,74 @@ def models():
+            try:
+                uid = get_value(request, 'data', 'uid')
+                SKETCHFAB_DOMAIN = 'sketchfab.com'
+                SKETCHFAB_API_URL = 'https://api.{}/v3'.format(SKETCHFAB_DOMAIN)
+                
+                
+                token = get_value(request, 'config', 'token')
+                headers = {'Authorization': 'Token {}'.format(token)}
+                model__endpoint = SKETCHFAB_API_URL + '/models/{}'.format(uid)
             
+
+                r = requests.get(model__endpoint, headers=headers)
+            except requests.exceptions.RequestException as e:
+                return jsonify({'requestException': e})
+            else:
+                response = r.json()
+                print(response)
+                return jsonify(response)
+
+@app.route('/me', methods=['POST', 'GET', 'DELETE'])
+def me():
+    try:
+        #Get the user information 
+        if request.method == 'GET':
+            try:
+                SKETCHFAB_DOMAIN = 'sketchfab.com'
+                SKETCHFAB_API_URL = 'https://api.{}/v3'.format(SKETCHFAB_DOMAIN)
+                MODEL_ENDPOINT1 = SKETCHFAB_API_URL + '/me'
+                MODEL_ENDPOINT2 = SKETCHFAB_API_URL + '/me/followers'
+                MODEL_ENDPOINT3 = SKETCHFAB_API_URL + '/me/collections'
+                MODEL_ENDPOINT4 = SKETCHFAB_API_URL + '/me/backgrounds'
+                MODEL_ENDPOINT5 = SKETCHFAB_API_URL + '/me/followings'
+                MODEL_ENDPOINT6 = SKETCHFAB_API_URL + '/me/subscriptions'
+                MODEL_ENDPOINT7 = SKETCHFAB_API_URL + '/me/models'
+                MODEL_ENDPOINT8 = SKETCHFAB_API_URL + '/me/likes'
+                MODEL_ENDPOINT9 = SKETCHFAB_API_URL + '/me/environments'
+                ALL_ENDPOINT = [MODEL_ENDPOINT1, MODEL_ENDPOINT2, MODEL_ENDPOINT3, MODEL_ENDPOINT4, MODEL_ENDPOINT5, MODEL_ENDPOINT6, MODEL_ENDPOINT7, MODEL_ENDPOINT8, MODEL_ENDPOINT9]
+                
+                config = json.loads(request.form.get('config'))
+                auth = config.get('auth')
+                token = auth.get('token')
+                headers = {'Authorization': 'Token {}'.format(token)}
+                resp_obj = {}
             
-            token = get_value(request, 'config', 'token')
-            headers = {'Authorization': 'Token {}'.format(token)}
-            users_endpoint = SKETCHFAB_API_URL + '/users/{}'.format(uid)
-        
-
-            r = requests.get(users_endpoint, headers=headers)
-        except requests.exceptions.RequestException as e:
-            return jsonify({'requestException': e})
-        else:
-            response = r.json()
-            print(response)
-            return jsonify(response)
+            for x in ALL_ENDPOINT:
+                    r = requests.get(x, headers=headers)
+                except requests.exceptions.RequestException as e:
+                    return jsonify({'requestException': e})
+                else:
+                    resp_obj.update({x: r.json()})
```

Comment:
you need to update `resp_obj` inside the for loop. Please test your code to make sure it is working properly before submitting a pull request!